xDiffCollectionTests/CollectionTestTests.swift

```diff
@@ -0,0 +1,166 @@
+//
+//  CollectionTestTests.swift
+//  CollectionTestTests
+//
+//  Created by Esteban Garro on 2018-06-05.
+//  Copyright © 2018 aldo. All rights reserved.
+//
+
+import XCTest
+@testable import xDiffCollection
+
+fileprivate let object1 = CollectionTestObjectMock(value:"apple")
+fileprivate let object2 = CollectionTestObjectMock(value:"ansi")
+fileprivate let object3 = CollectionTestObjectMock(value:"bacon")
+fileprivate let object4 = CollectionTestObjectMock(value:"cocoon")
+
+class CollectionTestTests: XCTestCase {
+    
+    var myCollection : DiffCollection<CollectionTestObjectMock>?
+    
+    override func setUp() {
+        super.setUp()
+
+        let f1 = DiffCollectionFilter<CollectionTestObjectMock>(name: "Starts with a", filter:{ s in
+            if(s.value.starts(with: "a")) {
+                return true
+            }
+            return false
+        })
+        
+        let f2 = DiffCollectionFilter<CollectionTestObjectMock>(name: "Starts with b", filter:{ s in
+            if(s.value.starts(with: "b")) {
+                return true
+            }
+            return false
+        })
+        
+        let f3 = DiffCollectionFilter<CollectionTestObjectMock>(name: "Starts with c", filter:{ s in
+            if(s.value.starts(with: "c")) {
+                return true
+            }
+            return false
+        })
+        
+        myCollection = DiffCollection(filters: [f1,f2,f3])
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+    
+    func test_Fillup_Collection() {
```

Comment:
Don't you think the code below contains the same repetitive code? May be we could abstract it and make the test case more readable? 
Same comment for the code below. 
Lemme know if you need help with that. Probably we could use the same approach that we started in mFind. (Tester and TestCases). What do you think?

Reply:
Yes, definitely. It takes a bit more work to write the tests that way, but definitely the result is much nicer.